winml/test/model/ort_value_helper.cpp

```diff
@@ -0,0 +1,68 @@
+#include "testPch.h"
+#include "ort_value_helper.h"
+using namespace winml;
+
+namespace OrtValueHelpers {
+template <ONNXTensorElementDataType T>
+winml::ITensor CopyTensorData(Ort::Value& val) {
+  using WinMLTensorKind = typename ONNXTensorElementDataTypeToWinMLTensorKind<T>::Type;
+  ITensor tensor = nullptr;
+  WINML_EXPECT_NO_THROW(tensor = WinMLTensorKind::Create(val.GetTensorTypeAndShapeInfo().GetShape()));
```

Comment:
Can we wrap only this part in the templated function? The rest works regardless of the tensor kind

Reply:
That will avoid generating all the code for each template specialization

Reply:
Fixed